proposals/exception-handling/Exceptions.md

```diff
@@ -398,48 +386,39 @@ document](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).
 
 #### Other Types
 
-##### exception_type
+##### tag_type
 
 We reserve a bit to denote the exception attribute:
 
 | Name      | Value |
 |-----------|-------|
 | Exception | 0     |
 
-Each exception type has the fields:
+Each tag type has the fields:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| `attribute` | `varuint32` | The attribute of an exception. |
+| `attribute` | `varuint32` | The attribute of a tag. |
```

Comment:
Obviously a separate issue, but I'm just noticing this while I'm here: I'd conservatively make this a `uint8` for the time being, since we do not know yet how exactly this is gonna be used in the future, and a zero `uint8` is still forward compatible with `varuint32`. That's what we did in other places anyway.

Reply:
I can do that as a followup PR. But why `uint8` better than `varuint32` here?

Reply:
It's both more conservative and faster to decode. No reason to bother with allowing multibyte representations of 0 before we even know that we'll need them.